OCast/Core/Utils/HttpService.swift

```diff
@@ -0,0 +1,80 @@
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+//
+//  HttpService.swift
+//  OCast
+//
+//  Created by Christophe Azemar on 12/12/2018.
+//
+
+import Foundation
+
+public typealias HttpResult = (_ response: HTTPURLResponse?, _ data: Data?, _ error: Error?) -> Void
+
+public class HttpService {
+    
+    public static func get(url: String, httpHeaders: [String : String]? = nil, withCompletionHandler handler: HttpResult?) {
```

Comment:
Add a HTTPMethod enum instead. And the method to launch the request can take by default the get enum value.

Reply:
enum added